src/commands/branches.ts

```diff
@@ -293,20 +293,48 @@ const create = async (
           if (!branch) {
             throw new Error('No default branch found');
           }
-          return { parent_id: branch.id };
+          return { parent_id: branch.id, protected: branch.protected };
         });
     }
 
     if (looksLikeLSN(props.parent)) {
-      return { parent_lsn: props.parent };
+      return props.apiClient
```

Comment:
I think if only lsn is specified, then branch would be created from the default branch from the point in time, specified by this lsn

Reply:
It's easier to calculate if the parent branch is protected after the new branch is created.
Changed the logic to output the warning after the branch creation and rolled back this particular change.